worker_node/job/confirm.js

```diff
@@ -0,0 +1,55 @@
+const Base = require('./base');
+const Ain = require('@ainblockchain/ain-js').default;
+
+class Confirm extends Base {
+  static configProps = [
+    'ainUrl',
+    'startBlockNumber',
+    'finishBlockNumber',
+    'txHashList',
+  ];
+  #ain;
```

Comment:
#ain = null; ?

Reply:
I used `#ain` as the meaning of declaration and initialized it in `constructor`. ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)) If I initialize it to `null`, the IDE infers the type to `null`. As a result it gives warnings like this 😢 
![image](https://user-images.githubusercontent.com/32831939/92922703-37086400-f471-11ea-8eea-55eb6cd15e5c.png)